styles/06-components/rating.css

```diff
@@ -0,0 +1,42 @@
+:root {
+  --color-rating-before: #7e7d7d;
+  --color-rating-after: #e7d10a;
+}
+
+.rating-wrapper{
+  direction: rtl;
+  text-align: center;
+  position: relative;
+  cursor: pointer;
+  margin: 5ch;
+}
+
+.rating-wrapper input[type="radio"] {
```

Comment:
is this being used to hide the radio button circle?

Reply:
Yes